### ISSUE TITLE
For new menu items, fabrik list options cannot be set

### ISSUE THREAD

**Issue by EricdeWaal** [joomlahenk/fabrik#549]
In an existing menu item, fabrik list options for 'Show in list' and 'Prefilter' work as expected.

For a new menu item, only a very small pop-up window is shown. If the + (which is not white but black) is clicked, no dropdown list with elements is loaded, just a  - sign. 
![afbeelding](https://user-images.githubusercontent.com/5586423/203301693-55f95bca-9846-4212-a451-aa60ad566a4c.png)
![afbeelding](https://user-images.githubusercontent.com/5586423/203307189-b9d16b98-77b6-4961-a147-7917d075877d.png)
![afbeelding](https://user-images.githubusercontent.com/5586423/203307650-df5b6c39-c5a4-4008-a1f1-5106e7aea5ee.png)
![afbeelding](https://user-images.githubusercontent.com/5586423/203309016-ae88bfd9-1524-4096-9e60-4ccf7bd2f941.png)

**Comment by EricdeWaal** [comment]
The same behaviour occurs with a Fabrik list in a module.

**Comment by skurvish** [comment]
Try saving the menu item first, then see if the popup populates correctly.

**Comment by EricdeWaal** [comment]
Yes indeed, then it works. I checked in Fabrik 3 and it behaves the same. Never noticed that.
That only leaves the black + sign then ... ;<)

**Comment by skurvish** [comment]
Try this. Unzip into web root. 
[issue549.zip](https://github.com/joomlahenk/fabrik/files/10068112/issue549.zip)

**Comment by EricdeWaal** [comment]
Yes that works, thanks!
